metascript/animation.py

```diff
@@ -19,25 +19,11 @@
 from datetime import datetime
 from math import floor
 
+TMP_ANIMATION_DIRECTORY = 'tmp/animation'
 PERFORMANCE_KEYWORD = 'performance:'
 
 
-def _generate_animation_recorder_vrml(duration, output, controller_name):
-    return f'''
-    DEF ANIMATION_RECORDER_SUPERVISOR Robot {{
-    name "animation_recorder_supervisor"
-    controller "animator"
-    controllerArgs [
-        "--duration={duration}"
-        "--output={output}"
-        "--controller={controller_name}"
-    ]
-    supervisor TRUE
-    }}
-    '''
-
-
-def record_animations(config, destination_directory, controller_name):
+def record_animations(config, destination_directory, controller_path):
```

Comment:
When record_animations() is called by `competition.py`, it is given `TMP_ANIMATION_DIRECTORY` as an argument. Now that it is defined in this file, we could replace all occurrences of `destination_directory` by `TMP_ANIMATION_DIRECTORY` and remove `destination_directory` from the arguments of this function